solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -1159,7 +1175,11 @@ describe("RandomBeacon - Group Creation", () => {
 
           await mineBlocks(params.dkgResultChallengePeriodLength)
 
-          await randomBeacon.approveDkgResult()
+          tx = await randomBeacon.approveDkgResult()
```

Comment:
`tx` is not used and I believe we can also get rid of `let tx: ContractTransaction`

Reply:
Or... was the intention to add a simple `it("should work` to this describe given that we changed `it("should succeed"` into `beforeEach`? I think adding check about this event is a good move. We could check there some really basic stuff, e.g. if `DkgResultApproved` and `GroupActivated` were emitted.

Reply:
Regarding the latter, it was not an intention. I just refactored that case to use `beforeEach` and be easily extensible with new `it` assertions. The basic stuff you mentioned is already checked in earlier cases above so checking them again seems redundant. Regarding usused `tx`, see: https://github.com/keep-network/keep-core/pull/2691/commits/56ead30afdaad1d6fce5f6a6d54e61ced13cbddb

Reply:
That "should suceed" we had before was a kind of a shortcut: we don't want to repeat all previous assertions but we also want to have an explicit assertion this transaction passes at `startBlock + dkgTimeout - 1` and is supposed to do everything else as in the case tested in the earlier context.

This is also tested here because `beforeEach` would fail but... I think we miss an explicit "should work" assertion that at least suggests we expect exactly the same behavior as before. I do agree repeating the same tests does not make sense but I'd add that "should work" with at least one or two events checked so that we are not only focused on unlocking the pool but also saying the result needs to be accepted at that time, just like in any other case.

Reply:
Done as part of https://github.com/keep-network/keep-core/pull/2691/commits/2a790b469e3d5dd9e857523f9be5b6f67ac3e9bc